library/src/test/java/com/novoda/downloadmanager/NetworkFileDownloaderTest.java

```diff
@@ -63,7 +67,12 @@ public void emitsError_whenResponseCodeIsInvalid() throws IOException {
 
         networkFileDownloader.startDownloading(ANY_RAW_URL, UNKNOWN_FILE_SIZE, callback);
 
-        verify(callback).onError();
+        String expectedCause = String.format(
+                "Request: %s with response code: %s failed.",
+                ANY_RAW_URL,
+                418
+        );
```

Comment:
Probably best to just define a constant string. Otherwise it feels like it's following the code from the test

Reply:
True 👍